quinn/src/tests.rs

```diff
@@ -532,6 +532,7 @@ fn run_echo(args: EchoArgs) {
             // platforms in the doc comments of the various `local_ip` functions.
```

Comment:
Please update docs as discussed in this above comment.

Reply:
Note these docs are being centralized in https://github.com/quinn-rs/quinn/pull/1868.